src/gt4py/__init__.py

```diff
@@ -37,6 +37,7 @@
 
 from .definitions import AccessKind, Boundary, DomainInfo, FieldInfo, ParameterInfo, CartesianSpace
 from .stencil_object import StencilObject
+from .test_builder import TestBuilder
```

Comment:
`TestBuilder` seems like something that should not be imported by default.

Reply:
Removed.